app/user/user.sidebar.js

```diff
@@ -9,20 +24,37 @@
   /**
    * @ngInject
    */
-  function NavConfig(NavProvider) {
-    NavProvider.group('user', {
+  function NavConfig(Auth, NavProvider, Permission) {
+    //Need to import Auth and Permission here. Why doesn't this work? in install.nav.config.js and search.tab.js
+    //this is used in seemingly the exact same way.
+    var group = NavProvider.group('user', {
```

Comment:
why doesnt this work? the other times auth is used it appears to be the exact same.